src/checkers/Cashflow.ml

```diff
@@ -1604,21 +1630,38 @@ module ScillaCashflowChecker
                   cf_tag_expr e (lub_tags (get_id_tag f) NoInfo) (AssocDictionary.make_dict ()) (AssocDictionary.make_dict ()) ctr_tag_map in
              AssocDictionary.insert (get_id f) e_tag acc_env)
       in
-      let rec tagger cconstraint components param_env field_env ctr_tag_map =
+    let rec tagger cconstraint (fields:(field list)) components param_env field_env ctr_tag_map =
```

Comment:
This type annotation isn't necessary. It should be fairly obvious what type `fields` has.